### DISCUSSION TITLE
consensus/Paxos-history

### DISCUSSION THREAD

**Discussion by giscus** [isno/theByteBook#202] [General]
# consensus/Paxos-history

构建大规模高可用的分布式系统

https://www.thebyte.com.cn/consensus/Paxos-history.html

**Comment by nieen** [comment]
本章节的章节号错了，当前是6.2章节下的6.3.1、6.3.2，应该是6.2.1、6.3.2

**Reply by nieen** [reply]
应该是6.2.1、6.2.2

**Reply by isno** [reply]
谢谢，已修正。最近正在校验内容，6月底，可以全部修正完。
